Marlin/src/MarlinCore.cpp

```diff
@@ -937,14 +937,6 @@ void setup() {
 
   SETUP_RUN(HAL_init());
 
-  // Init and disable SPI thermocouples
-  #if HEATER_0_USES_MAX6675
-    OUT_WRITE(MAX6675_SS_PIN, HIGH);  // Disable
-  #endif
-  #if HEATER_1_USES_MAX6675
-    OUT_WRITE(MAX6675_SS2_PIN, HIGH); // Disable
-  #endif
-
   #if HAS_L64XX
```

Comment:
Question: the following **_block use to be needed to prevent an SPI conflict_** on startup for the MAX31855 boards when the HAL manipulates the PINS manually:
```
  // Init and disable SPI thermocouples
  #if HEATER_0_USES_MAX6675
    OUT_WRITE(MAX6675_SS_PIN, HIGH);  // Disable
  #endif
  #if HEATER_1_USES_MAX6675
    OUT_WRITE(MAX6675_SS2_PIN, HIGH); // Disable
  #endif
```
**Has the SPI conflict been resolved** for the MAX31855 thermocouples where an external LIB call is not used (but the HALs use there own SPI interface calls)[example, GTR V1.0 board has a built in MAX31855 board but Marlin does not use a external LIB call to obtain Software SPI or Hardware SPI for the MAX31855 temperature Sensor]?

Since this PR does not remove Marlin SW capability to use a MAX31855 board WITHOUT a LIB call, the following block of statements are still needed (unless the SPI conflict at startup has been resolved) because the HALs are manipulating the max_tc_spi interface by manual bit banging to obtain Software SPI and Hardware SPI.

The below sections are what I call "the HALs obtaining the MAX31855 temperature via manual bit banging to obtain Software SPI and Hardware SPI WITHOUT the use of an external Library call":
```
SPIclass<MAX6675_DO_PIN, SD_MOSI_PIN, MAX6675_SCK_PIN> max_tc_spi
```
```
TERN_(THERMO_SEPARATE_SPI, max_tc_spi.init());

and

    #if !THERMO_SEPARATE_SPI && NO_THERMO_TEMPS
      spiBegin();
      spiInit(MAX_TC_SPEED_BITS);
    #endif

    #if NO_THERMO_TEMPS
      MAX6675_WRITE(LOW);  // enable TT_MAX6675
      DELAY_NS(100);       // Ensure 100ns delay
    #endif

    max_tc_temp = 0;

    // Read a big-endian temperature value
    #if NO_THERMO_TEMPS
      for (uint8_t i = sizeof(max_tc_temp); i--;) {
        max_tc_temp |= TERN(THERMO_SEPARATE_SPI, max6675_spi.receive(), spiRec());
        if (i > 0) max_tc_temp <<= 8; // shift left if not the last byte
      }
        MAX6675_WRITE(HIGH); // disable TT_MAX6675
    #endif

```

Reply:
> Has the SPI conflict been resolved for the MAX31855 thermocouples where an external LIB call is not used

As far as I know it has not been resolved.